scripts/Fig3D_merged-umaps.R

```diff
@@ -0,0 +1,90 @@
+# This script is used to generate a small version of the merged UMAP for Figure 3
+
+# load project
+renv::load()
+
+library(SingleCellExperiment)
+library(dplyr)
+library(ggplot2)
+
+# Set default ggplot theme
+theme_set(
+  theme_classic() +
+    theme(
+      #plot.margin = margin(rep(20, 4)),
+      strip.background = element_rect(fill = "transparent", linewidth = 0.5),
+      # no axis ticks or labels
+      axis.line = element_blank(),
+      axis.ticks = element_blank(),
+      axis.text = element_blank(),
+      # add a square around each of the plots
+      panel.background = element_rect(colour = "black", linewidth=0.5),
+      aspect.ratio = 1
+    )
+)
+
+set.seed(2024)
```

Comment:
not needed unless we are _calculating_ the UMAP, which we're not!

Reply:
Sorry this was leftover from copying this set up chunk, I'll remove it.